code/game/objects/items/charter.dm

```diff
@@ -60,6 +60,10 @@
 	// Autoapproves after a certain time
 	response_timer_id = addtimer(CALLBACK(src, .proc/rename_station, new_name, user.name, user.real_name, key_name(user)), approval_time, TIMER_STOPPABLE)
 	to_chat(GLOB.admins, "<span class='adminnotice'><b><font color=orange>CUSTOM STATION RENAME:</font></b>[ADMIN_LOOKUPFLW(user)] proposes to rename the [name_type] to [new_name] (will autoapprove in [DisplayTimeText(approval_time)]). [ADMIN_SMITE(user)] (<A HREF='?_src_=holder;[HrefToken(TRUE)];reject_custom_name=[REF(src)]'>REJECT</A>) [ADMIN_CENTCOM_REPLY(user)]</span>")
+	for(var/client/X in GLOB.admins)
```

Comment:
Better variable name please. Something like `admin_client`, or even just `admin` if its not taken.

Reply:
I mean, it's pretty obvious what it is and it's the only place in the whole file where it's used. Does it really need a snowflake var?

Reply:
Yeah, single letter variables just tend to be a code smell, I think we're slowly weeding them out.